curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-oop-by-building-a-platformer-game/65b2bb4c279af3cd585ba777.md

```diff
@@ -21,10 +21,10 @@ You should have a checkpoint detection rule that checks for the following: `play
 assert.match(code, /player\.position\.x\s*-\s*player\.width\s*<=\s*checkpoint\.position\.x\s*-\s*checkpoint\.width\s*\+\s*player\.width\s*\*\s*0\.9/i);
 ```
 
-You should have a checkpoint detection rule that checks for the following: `index === 0 || checkpoints[index - 1].claimed === true`.
+You should have a checkpoint detection rule that checks for the following: `index === 0 || checkpoints[index - 1].claimed === true` or `index === 0 || checkpoints[index - 1].claimed`.
 
 ```js
-assert.match(code, /index\s*===\s*0\s*\|\|\s*checkpoints\[index\s*-\s*1\]\.claimed\s*===\s*true/i);
+assert.match(code, /index\s*===\s*0\s*\|\|\s*checkpoints\[index\s*-\s*1\]\.claimed(?!\s*===\s*false)/i);
```

Comment:
same note here

Let's revert this change for now. 
It is outside of the scope of this particular PR.